includes/Stories_Renderer/Renderer.php

```diff
@@ -488,9 +488,7 @@ public function render_single_story_content() {
 				class="<?php echo esc_attr( $single_story_classes ); ?>"
 				on="<?php echo esc_attr( sprintf( 'tap:AMP.setState({%1$s: ! %1$s})', $lightbox_state ) ); ?>"
 			>
-				<?php
-				$this->render_story_with_poster();
-				?>
+				<?php $this->render_story_with_poster(); ?>
```

Comment:
Naming of this method doesn't really make sense anymore.

Reply:
Perhaps it's not ideal, but I wouldn't really change it now at this point.

Reply:
> I wouldn't really change it now at this point. 

Why not? In PHPStorm, I can change it will update all the references.

Reply:
I know it can be done, I just don't think we should.

The name isn't so bad that it warrants a renaming just because there's now some handling for when there is no poster.

I also don't think we should split this up into two methods or something.

If anything I'd just remove the method and move everything to `render_single_story_content`.

But it's still not really worth it IMO.